apps/app.go

```diff
@@ -135,7 +136,7 @@ func marathonAppNameToServiceName(name string, nameSeparator string) string {
 }
 
 func labelsToTags(labels map[string]string) []string {
-	tags := []string{}
+	tags := make([]string, 0, len(labels))
```

Comment:
`len(labels)` is ineffective, the first thing that will happen to it is `append`, which will allocate new slice to fit `tags` and `commonTags`.